web/impact/impact/tests/test_algolia_api_key_view.py

```diff
@@ -122,6 +126,30 @@ def test_user_with_staff_role_grant_sees_all_mentors(self):
                 response_data = json.loads(response.content)
                 self.assertEqual(response_data["filters"], [])
 
+    def test_staff_user_does_not_have_is_active_filter(self):
```

Comment:
@mkiterian there seems to be some duplication in the tests, refactor the common pieces into a different function/method.